src/Compilers/CSharp/Portable/CodeGen/EmitStatement.cs

```diff
@@ -12,6 +12,7 @@
 using Microsoft.CodeAnalysis.PooledObjects;
 using Microsoft.CodeAnalysis.Text;
 using Roslyn.Utilities;
+using static Microsoft.CodeAnalysis.CSharp.Binder;
```

Comment:
Why is this `using static` needed?

Reply:
This code uses `AddressKind` a lot, which has been moved.